auth/facebook.js

```diff
@@ -0,0 +1,63 @@
+import passport from 'passport';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+import { User } from '../models';
+
+passport.use(new FacebookStrategy({
+  clientID: process.env.FACEBOOK_CLIENT_ID,
+  clientSecret: process.env.FACEBOOK_CLIENT_SECRET,
+  callbackURL: 'http://localhost:7000/api/v1/auth/facebook/callback',
```

Comment:
@Dubby20  remember to change this callbackURL

Reply:
Alright I will but  it has to be on Staging branch